Dockerfile

```diff
@@ -1,51 +1,45 @@
-FROM ubuntu:22.04
+FROM debian:bookworm
 
 ENV DEBIAN_FRONTEND=noninteractive
 
 # Install system packages then clean up to minimize image size
 RUN apt-get update \
- && apt-get install --no-install-recommends -y \
-      build-essential \
-      curl \
-      default-jre-headless \
-      file \
-      git-core \
-      gpg-agent \
-      libarchive-dev \
-      libffi-dev \
-      libgd-dev \
-      libpq-dev \
-      libsasl2-dev \
-      libvips-dev \
-      libxml2-dev \
-      libxslt1-dev \
-      libyaml-dev \
-      locales \
-      postgresql-client \
-      ruby \
-      ruby-dev \
-      ruby-bundler \
-      software-properties-common \
-      tzdata \
-      unzip
-
-# Install Node.js 18 and npm
-RUN curl -fsSL https://deb.nodesource.com/setup_18.x | bash - \
- && apt-get install -y nodejs
+  && apt-get install --no-install-recommends -y \
+  build-essential \
+  curl \
+  default-jre-headless \
+  file \
+  git-core \
+  gpg-agent \
+  libarchive-dev \
+  libffi-dev \
+  libgd-dev \
+  libpq-dev \
+  libsasl2-dev \
+  libvips-dev \
+  libxml2-dev \
+  libxslt1-dev \
+  libyaml-dev \
+  locales \
+  postgresql-client \
+  ruby \
+  ruby-dev \
+  ruby-bundler \
+  software-properties-common \
+  tzdata \
+  unzip \
+  nodejs \
+  npm \
+  osmosis
 
 # Install yarn globally
-RUN npm install --global yarn \
- # We can't use snap packages for firefox inside a container, so we need to get firefox+geckodriver elsewhere
- && add-apt-repository -y ppa:mozillateam/ppa \
- && echo "Package: *\nPin: release o=LP-PPA-mozillateam\nPin-Priority: 1001" > /etc/apt/preferences.d/mozilla-firefox \
- && apt-get install --no-install-recommends -y \
-      firefox-geckodriver \
- && apt-get clean \
- && rm -rf /var/lib/apt/lists/*
-
-# Install compatible Osmosis to help users import sample data in a new instance
-RUN curl -OL https://github.com/openstreetmap/osmosis/releases/download/0.47.2/osmosis-0.47.2.tgz \
- && tar -C /usr/local -xzf osmosis-0.47.2.tgz
+RUN npm install --global yarn
+
+# Install geckodriver and firefox
+RUN apt-get update                             \
+  && apt-get install -y --no-install-recommends \
+  ca-certificates curl firefox-esr           \
```

Comment:
These installs can be done with all the others now - they were only separate because we needed to install `software-properties-common` first to get `add-apt-repository` which was then used here.

Reply:
Great catch, moved that so they are installed with others